core/src/main/scala/org/apache/spark/status/protobuf/AccumulableInfoSerializer.scala

```diff
@@ -25,7 +25,7 @@ import org.apache.spark.status.api.v1.AccumulableInfo
 import org.apache.spark.status.protobuf.Utils.{getOptional, getStringField, setStringField}
 import org.apache.spark.util.Utils.weakIntern
 
-private[protobuf] object AccumulableInfoSerializer {
+private[spark] object AccumulableInfoSerializer {
```

Comment:
cc @gengliangwang @dongjoon-hyun FYI

Do you prefer `private[spark]` or  `private[protobuf]`?

https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/status/protobuf/ProtobufSerDe.scala

Should `ProtobufSerDe`  also be changed to `private[spark]`? Although it has been identified as `@DeveloperApi`

Reply:
Developer APIs are fine.

> Do you prefer private[spark] or private[protobuf]?

Whichever narrower should be preferred.

Reply:
OK, Let me change to `private[protobuf]`